server/remote_cache/digest/digest.go

```diff
@@ -62,6 +62,17 @@ func NewResourceName(d *repb.Digest, instanceName string) *ResourceName {
 	}
 }
 
+func NewCacheResourceName(d *repb.Digest, instanceName string, cacheType rspb.CacheType) *ResourceName {
```

Comment:
You could make life slightly easier for callers by having a `NewCASResourceName` and `NewACResourceName` funcs. Slightly more concise.

Reply:
Ack